utils/funcexp/func_cast.cpp

```diff
@@ -118,7 +118,8 @@ int64_t Func_cast_signed::getIntVal(Row& row, FunctionParm& parm, bool& isNull,
     case execplan::CalpontSystemCatalog::UTINYINT:
     case execplan::CalpontSystemCatalog::USMALLINT:
     {
-      return (int64_t)parm[0]->data()->getUintVal(row, isNull);
+      const auto val = static_cast<int64_t>(parm[0]->data()->getUintVal(row, isNull));
```

Comment:
I assume you want to apply the defined methods here?

Reply:
I didn't implement any function for converting IntValues to MCS_BIGINT values. Felt unnecessary if we can also make the logic clear here - do you I think I should also extract this logic to a method?

Reply:
If you do the logic clear just here there might be another place where the same functionality is needed, namely to bound the impossible NULL value. IMHO it is better to put it into a method and use here.